src/progaudi.tarantool/FeatureExtensions.cs

```diff
@@ -0,0 +1,15 @@
+﻿using ProGaudi.Tarantool.Client.Model;
+
+namespace ProGaudi.Tarantool.Client
+{
+    public static class FeatureExtensions
+    {
+        public static bool IsSqlAvailable(this BoxInfo info)
```

Comment:
Do we really need it as extension?
Can we place it inside SqlAvailable property logic? Or inside BoxInfo if it will be used somewhere else?

Reply:
What is your proposition?

No, this should be in an extension to BoxInfo. Anyone can use it, since it is public extension.

Reply:
According to [Microsoft guidelines](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/extension-methods) you should place it inside BoxInfo.